src/Adapter/Product/AdminProductWrapper.php

```diff
@@ -76,11 +84,15 @@ class AdminProductWrapper
      * Constructor : Inject Symfony\Component\Translation Translator.
      *
      * @param object $translator
+     * @param LegacyContext $legacyContext
+     * @param LocaleRepository $localeRepository
+     * @param string $locale
      */
-    public function __construct($translator, $legacyContext)
+    public function __construct($translator, $legacyContext, LocaleRepository $localeRepository, string $locale)
```

Comment:
You can only inject the locale here. Ideally we could even remove the context and inject the employee shop list but let's calm down a bit 😅

Reply:
Double Fixed